### PR TITLE
libbi 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by sbfnk**
### Have you:

- [X] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [X] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [X] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [X] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [X] Written a sensible test? (The best test is to compile and run a sample program.)

State-space modelling and Bayesian inference on high-performance computer hardware, including multi-core CPUs, many-core GPUs (graphics processing units) and distributed-memory clusters.

### CODE REVIEW

**Comment by sbfnk** [comment]
Checks failing with

```
Error: 1 problem in 1 formula
homebrew/science/libbi:
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
```

File: libbi.rb [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  end
+
+  resource "thrust" do
+    url "https://github.com/thrust/thrust/releases/download/1.8.2/thrust-1.8.2.zip"
+    sha256 "00925daee4d9505b7f33d0ed42ab0de0f9c68c4ffbe2a41e6d04452cdee77b2d"
+  end
+
+  def install
+    ENV.prepend_path "PATH", Formula["perl"].bin
+    ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
```
Comment (jonchang) [inline libbi.rb]: See comment above re: system vs brewed Perl

File: libbi.rb [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  head "https://github.com/libbi/LibBi.git"
+
+  option "without-test", "Disable build-time checking (not recommended)"
+
+  depends_on "perl"
+  depends_on "homebrew/science/qrupdate"
+  depends_on "homebrew/science/netcdf"
+  depends_on "gsl"
+  depends_on "boost"
+  depends_on "automake"
```
Comment (sbfnk) [inline libbi.rb]: On this occasion, no, because automake is also needed at runtime (when models are parsed and compiled).

File: libbi.rb [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+  # disable openmp, currently not working on OSX
+  patch :DATA
```
Comment (sbfnk) [inline libbi.rb]: Thanks, it does indeed work with openMP. This is now optional (enabled by default).

File: libbi.rb (hunk 1 above)
Comment (sbfnk) [inline libbi.rb]: I removed the PATH line.

File: libbi.rb (hunk 3 above)
Comment (sbfnk) [inline libbi.rb]: Note that this did require including two patches as otherwise it wouldn't (always) compile if CUDA is not installed.

**Comment by jonchang** [comment]
Thanks @sbfnk!

**Pull request closed without merging**